programming/vision/index.md

```diff
@@ -34,7 +34,10 @@ while True:
     print("I can see", len(markers), "markers:")
 
     for marker in markers:
-        print(" - Marker #{0} is {1} metres away".format(marker.id, marker.distance / 1000))
+        print("Marker #{0} is {1} metres away".format(
+            marker.id,
+            marker.position.distance / 1000,
+        ))
```

Comment:
I'm assuming it's the `.position` part which was missing? (Reflowing the code at the same time as fixing this and also changing the template makes it somewhat harder to spot the difference)